app/models/orchestration_template.rb

```diff
@@ -144,6 +144,18 @@ def save_as_orderable!
     new_record? ? replace_with_old_template(old_template) : transfer_stacks(old_template)
   end
 
+  def deep_copy(new_attributes = {})
```

Comment:
not really sure this deep copy is needed since it's only dup-ing and updating the attributes